papiea-engine/src/utils/utils.ts

```
@@ -70,4 +70,12 @@ export function isEmpty(obj: any) {
             return false;
     }
     return true;
+}
+
+export function safeJSONParse(chunk: string) {
+    try {
+        return JSON.parse(chunk)
+    } catch (e) {
+        return undefined
```

Should we just silently ignore errors? Can't this hide real problems?